### ISSUE TITLE
Comparison with bigdata:visits-tracker

### ISSUE THREAD

**Issue by dandv** [usefulio/visitor#3]
Hi Ian,

There's already a package for tracking visitors, including signed in ones, @andrewreedy's https://atmospherejs.com/bigdata/visit-tracker. Wondering how this one compares - would be nice to mention that in the README and help devs make an informed decision.

Thanks!

**Comment by ianserlin** [comment]
Good idea, we can definitely learn from each other's approaches. I can't promise when I'll do this, but I'm leaving it open with the intention of it being done.

**Comment by SevenZark** [comment]
Considering that bigdata's visit-tracker repo currently returns a 404 here in github, this library may now be just what we need. :)

**Comment by andrewreedy** [comment]
Yea I rm'd that repo it was unmaintained and outdated.
